core/server/master/src/test/java/alluxio/master/metastore/heap/HeapBlockStoreTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.metastore.heap;
+
+import static org.junit.Assert.assertEquals;
+
+import alluxio.proto.meta.Block;
+
+import org.junit.Test;
+
+public class HeapBlockStoreTest {
```

Comment:
it looks to me `HeapBlockStoreTest#blockSize` is identical to  `RocksBlockStoreTest#blockSize`

How about having a new  parametered test `BlockStoreTest`, with blockStore to be both `RocksBlockStore` and `HeapBlockStore`? check alluxio/client/file/cache/store/PageStoreTest.java as reference